Fix end date for last periods (#6584)
woocommerce/wc-admin: TESTING-INSTRUCTIONS.md

## Unreleased

### Fix end date for last periods #6584

> 1. We need to be in March 2021

Reviewer: This may be a little tricky to test.  Maybe we could update this to:

```suggestion
1. Update your system clock to March 2021
```

Author: Thank you for the feedback. The changes has been pushed.

3. Place an order with the product
4. Navigate to Analytics -> Categories
5. You should see the category.

> >>>>>>> cdc813055 (Add changelog)

Reviewer: Thanks for the fix!
